Formula/minicom.rb

```diff
@@ -20,15 +20,22 @@ class Minicom < Formula
     sha256 x86_64_linux:   "7d1b0aae1f169968d42e4dea644dff5a4f18010b59b334439aa2bd276c6e913a"
   end
 
+  head do
+    url "https://salsa.debian.org/minicom-team/minicom.git", branch: "master"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "gettext"
```

Comment:
We don't want to add this dependency to the stable build, especially since it wasn't there before.

Reply:
I might be wrong (again) but following the cookbook https://docs.brew.sh/Formula-Cookbook#double-check-for-dependencies I have for 2.8 as well as HEAD:
```bash
$ otool -L src/minicom
src/minicom:
	/usr/local/opt/gettext/lib/libintl.8.dylib (compatibility version 11.0.0, current version 11.0.0)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1858.112.0)
	/usr/lib/libiconv.2.dylib (compatibility version 7.0.0, current version 7.0.0)
	/usr/lib/libncurses.5.4.dylib (compatibility version 5.4.0, current version 5.4.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.100.3)
```
I'm not sure `minicom` nor the build will succeed without `gettext`

Reply:
That looks like opportunistic linkage to me. We built it without `gettext` before, so it would be weird if it fails to build now without it.

Reply:
Ok, got it, the `ABOUT-NLS` file states this:

>    When you already have GNU `gettext` installed on your system and run
configure without an option for your new package, `configure` will
probably detect the previously built and installed `libintl.a` file and
will decide to use this.  This might not be desirable.  You should use
the more recent version of the GNU `gettext` library.  I.e. if the file
`intl/VERSION` shows that the library which comes with this package is
more recent, you should use
     `./configure --with-included-gettext`
to prevent auto-detection.

So I guess I can safely remove this dependency indeed, thank you